pallets/funding/src/tests/misc.rs

```diff
@@ -191,6 +214,8 @@ mod helper_functions {
 		let returned_plmc_balances = returned_plmc_mappings.into_iter().map(|map| map.plmc_amount).collect_vec();
 
 		for (expected, calculated) in zip(expected_returns, returned_plmc_balances) {
+			let expected = FixedU128::from_float(expected);
+			let expected = expected.checked_mul_int(PLMC).unwrap();
 			assert_close_enough!(expected, calculated, Perquintill::from_float(0.99));
```

Comment:
```suggestion
for (expected_return, returned_balance) in zip(expected_returns, returned_plmc_balances) {
    let expected_value = FixedU128::from_float(expected_return)
        .checked_mul_int(PLMC)
        .unwrap();

    assert_close_enough!(expected_value, returned_balance, Perquintill::from_float(0.99));
}
```

nit: can we avoid shadowing of variables?